ci-operator/step-registry/ipi/conf/azure/resourcegroup/ipi-conf-azure-resourcegroup-commands.sh

```diff
@@ -10,7 +10,15 @@ curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -
 CONFIG="${SHARED_DIR}/install-config.yaml"
 PATCH="/tmp/install-config-existingresourcegroup.yaml.patch"
 
-azure_region=$(/tmp/yq r "${CONFIG}" 'platform.azure.region')
+if [[ ! -z ${AZURE_REGION:-} ]]; then
```

Comment:
No, this is not how prow ci work. If you want  a random region selected by prow ci, they are controlled by `azure-qe-quota-slice` defined in `core-services/prow/02_config/_boskos.yaml`